hhvm-serve.go

```diff
@@ -1,10 +1,10 @@
 package main
 
-import "github.com/beberlei/hhvm-serve/fcgiclient"
+import "github.com/LionsAd/hhvm-serve/fcgiclient"
```

Comment:
What New method? Can't see any, if you could make the change that would be awesome. Ping me here and I can merge afterwards.